src/geolocation/component.js

```diff
@@ -88,6 +89,7 @@ function geolocationComponent() {
     restrict: 'A',
     scope: {
       'getMapFn': '&ngeoGeolocationMap',
+      'getMainCtrlFn': '&ngeoGeolocationMainctrl',
```

Comment:
Directling passing ``mainCtrl.loading`` instead would make sense. I'm looking for a way to watch this attribute from a distinct controller using some sync mechanism...